diff --git a/src/str.c b/src/str.c
index 16a4bc4..e04b0c9 100644
--- a/src/str.c
+++ b/src/str.c
@@ -35,12 +35,12 @@ Err mem_get_indexes(char* data, size_t len, size_t offset, char c, ArlOf(size_t)
 }
 
 
-inline const char* cstr_skip_space(const char* s) {
+const char* cstr_skip_space(const char* s) {
     for (; *s && isspace(*s); ++s);
     return s;
 }
 
-inline const char* cstr_next_space(const char* l) {
+const char* cstr_next_space(const char* l) {
     while (*l && !isspace(*l)) { ++l; }
     return l;
 }
diff --git a/src/wrapper-lexbor.c b/src/wrapper-lexbor.c
index 940659b..7d5b198 100644
--- a/src/wrapper-lexbor.c
+++ b/src/wrapper-lexbor.c
@@ -2,6 +2,10 @@
 #include "src/htmldoc.h"
 #include "src/wrapper-lexbor.h"
 
+/* This include seems to be required for termux build
+ * because the compiler in that platform needs to know
+ * the size of struct lxb_dom_text.  */
+#include <lexbor/dom/interfaces/text.h>
 //lxb_inline lxb_status_t append_to_buf_callback(const lxb_char_t *data, size_t len, void *bufptr) ;
 
 /* internal linkage */
@@ -272,3 +276,12 @@ Err dbg_print_title(lxb_dom_node_t* title) {
     fwrite("\n", 1, 1, stdout);
     return Ok;
 }
+
+Err lexbor_node_get_text(lxb_dom_node_t* node, const char* data[static 1], size_t len[static 1]) {
+    lxb_dom_text_t* text = lxb_dom_interface_text(node);
+    if(!text) return "error: expecting not null lxb_dom_interface_text(node)";
+    *data = (const char*)text->char_data.data.data;
+    *len = text->char_data.data.length;
+    return Ok;
+}
+
diff --git a/src/wrapper-lexbor.h b/src/wrapper-lexbor.h
index 8a24442..5e1765c 100644
--- a/src/wrapper-lexbor.h
+++ b/src/wrapper-lexbor.h
@@ -44,14 +44,7 @@ Err mk_submit_url (
     HttpMethod doc_method[static 1] 
 );
 
-static inline Err
-lexbor_node_get_text(lxb_dom_node_t* node, const char* data[static 1], size_t len[static 1]) {
-    lxb_dom_text_t* text = lxb_dom_interface_text(node);
-    if(!text) return "error: expecting not null lxb_dom_interface_text(node)";
-    *data = (const char*)text->char_data.data.data;
-    *len = text->char_data.data.length;
-    return Ok;
-}
+Err lexbor_node_get_text(lxb_dom_node_t* node, const char* data[static 1], size_t len[static 1]);
 
 static inline bool lexbor_inside_tag(lxb_dom_node_t* node, lxb_tag_id_enum_t tag) {
     while (node->parent) {
